doc/article/technical_article.tex

```diff
@@ -325,33 +320,34 @@ \subsection{User Interface}
 
 \section{Outlook}
 By focusing on doing one thing and doing it right, it was possible to create a
-modular platform. That did not just allow a clean process during development so
-far, but also enables open source contributors to easily and properly add
-features in the future. In the first place, the application needs some real life
-usage in order to smooth the edges and fulfill the promise of being simple and
-fast to use. Nevertheless, some bigger feature sets were thought up that could
-not be implemented yet.
+modular platform. This did not just allow a clean development process, but also
+enables open-source contributors to easily extend the trckr application. The
+main focus now should be to gather real life user experience and feedback to
+determine which features should be added next or be improved upon.
 
-Project focused features can be improved to enable more specific project
-methods, maybe even interface with ticketing or project management tools. This
-enables the team leaders to gain more insight on the progress.
+Project-focused features can be improved to enable more specific project
+methods, maybe even an interface with ticketing or project management tools.
+This enables the team leaders to gain more insight on the progress.
 
 A desktop application could be developed to automatically track usage of certain
-applications, or integrations for IDEs to track time worked on a project. By
-reducing the manual intervention by the user, the tracking could become even
-more accurate. This might even uncover certain inefficiencies, say by detecting
-too much time waiting for tests or deployments.
+applications or integrations for IDEs to track time worked on a project. By
```

Comment:
Do we have a glossary explaining the term IDE? otherwise we should put the written out meaning in parantheses, so it is explained.